src/test/resources/processExecutionsRecordings/react-native/run-android/win10-rn0.21.0/succedsWithTenVSEmulators.json

```diff
@@ -0,0 +1,850 @@
+{
+    "title": "react-native run-android runs succesfully when ten VS Emulators are online",
+    "composedFrom": "succedsWithFiveVSEmulators",
```

Comment:
It's not. All the metadata is informational at the moment.
I just wanted to say which was the original file that I modified to create this test, given that it's synthetic and not recorded.